sunpy/map/tests/test_mapbase.py

```diff
@@ -175,6 +175,12 @@ def test_date_aia(aia171_test_map):
     assert aia171_test_map.date == parse_time('2011-02-15T00:00:00.34')
 
 
+def test_date_setter(generic_map):
+    dtime_old = generic_map.date
+    generic_map.date = '2011-02-15T00:00:00.34'
+    assert generic_map.date == parse_time('2011-02-15T00:00:00.34') != dtime_old
```

Comment:
```suggestion
    new_date = '2011-02-15T00:00:00.34'
    generic_map.date = new_date
    assert generic_map.date == parse_time(new_date) != dtime_old
```
sorry for my pedancity of becoming DRY-er. This way we can change the date or parametrise it to use multiple range of dates if we need.